shared/lib_battery_voltage.cpp

```diff
@@ -275,8 +275,8 @@ void voltage_dynamic_t::initialize() {
 }
 
 voltage_dynamic_t::voltage_dynamic_t(int num_cells_series, int num_strings, double voltage, double Vfull,
-                                     double Vexp, double Vnom, double Qfull, double Qexp, double Qnom,
-                                     double C_rate, double R, double dt_hr) :
+                                     double Vexp, double Vnom, double Vcut, double Qfull, double Qexp, double Qnom,
+                                     double C_rate, double R, double dt_hr ) :
```

Comment:
I would put the Vcut after Qnom since it's not part of the exp, nom, full pairs between V and Q.

Reply:
Moved Vcut after Qnom, fixed in all voltage_dynamic_t calls.